src/main/java/no/unit/nva/model/Agent.java

```diff
@@ -0,0 +1,11 @@
+package no.unit.nva.model;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+@JsonSubTypes({
+    @JsonSubTypes.Type(name = "Organization", value = Organization.class)
+})
+public interface Agent {
```

Comment:
What is an Agent? Is it the institution through which the Resource has been published?